src/declarativedbusinterface.cpp

```diff
@@ -426,6 +428,10 @@ void DeclarativeDBusInterface::pendingCallFinished(QDBusPendingCallWatcher *watc
 
     if (reply.isError()) {
         qmlInfo(this) << reply.error();
+        QJSValue errorCallback = callbacks.second;
+        if (errorCallback.isCallable()) {
+            errorCallback.call();
```

Comment:
Could check whether the call succeeds: QJSValue foo errorCallback.call(); if (foo.isError()) { /* log error */ }